content/docs/instrumenting/http_sd.md

```diff
@@ -36,7 +36,7 @@ refresh interval.
 The SD endpoint must answer with an HTTP 200 response, with the HTTP Header
 `Content-Type: application/json`. The answer must be UTF-8 formatted.
 If no targets should be transmitted, HTTP 200 must also be emitted, with
-an empty list `[]`.
+an empty list `[]`. The list of unordered.
```

Comment:
I would call it target list, but nm.

`s/of/is/` and good to merge.

Reply:
What about Lists are unordered ?

Because this is a list of  maps that contain a target list.

Reply:
We're on the long tail of the discussion; "Target lists are unordered" would be even more generic; I am fine with any.